app.py

```diff
@@ -1,20 +1,135 @@
 from flask import Flask, render_template, request, redirect, url_for
+import config as cf
+from model import Item
+
 import session_items as session
+import requests
+import json
+
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
+#load_dotenv()
 
 @app.route('/')
 def index():
-    items = session.get_items()
-    return render_template('index.html', items=items)
-    #return 'Hello World!'
+    url = "https://api.trello.com/1/boards/{id}/cards"
+    query = cf.get_trello_query()
+    
+    response = requests.request(
+        "GET",
+        url,
+        params=query
+        )
+    #print(response.text)
+    #items = session.get_items()
+    #return render_template('index.html', items=items)
+    cards = json.loads(response.text)
+    items_list = list()
+    for card in cards:
+        item = Item(card.id, card.status, card.title)
+        items_list.append(item)
 
-@app.route('/add', methods=['POST'])
+    return render_template('index.html', items=items_list)
+
+# To Be Removed
+# @app.route('/add', methods=['POST'])
 def addToDo():
     title = request.form['title']
     session.add_item(title)
     return redirect(url_for('index'))
 
 if __name__ == '__main__':
     app.run()
+
+
+"""
+Add a complete_item route that accepts the ID of an item as a
+parameter and then calls a method to change its status from 'To
+Do' to 'Done'
+curl -i "localhost:5000/api/foo?a=hello&b=world" 
+"""
+@app.route('/complete_item/<idCard>', methods=['PUT'])
+def update_card(idCard):
+
+    add_card_to_done()
+
+    url = "https://api.trello.com/1/cards/{idCard}"
+    headers = {"Accept": "application/json"}
+    query = cf.get_trello_query()
+    #No obvious way of retrieving the done List Id, 
+    #therefore reusing/mocking the example one instead 
+    query['idList'] = '5abbe4b7ddc1b351ef961415'
+    response = requests.request( "PUT", url, headers=headers, params=query )
+    print(json.dumps(json.loads(response.text), sort_keys=True, indent=4, separators=(",", ": ")))
+
+
+#create card/item
+@app.route('/add', methods=['POST'])
+def add_card(list_id):
```

Comment:
Where are you expecting list_id to come from when you make a POST request to localhost:5000/add?

If you want to be able to choose what list to put a new card in, it should be part of the form.
Otherwise it should be from config.